blocks/blocks.go

```diff
@@ -148,20 +148,11 @@ func saveMetrics(blk types.Block) {
 			hist:  metrics.ForDiffLength,
 			value: float64(len(blk.ForDiff)),
 		},
-		{
-			hist:  metrics.NeutralDiffLength,
-			value: float64(len(blk.NeutralDiff)),
-		},
-		{
-			hist:  metrics.AgainstDiffLength,
-			value: float64(len(blk.AgainstDiff)),
-		},
 	}
 
 	for _, m := range metricList {
 		m.hist.
 			With(metrics.LayerIDLabel, blk.LayerIndex.String()).
```

Comment:
after some sought i am not sure that we need layer. why it would be useful to have a hist per layer? i think metrics should give a high level view into the network and system state. and if we need to find details about a particular blocks we can add those details to logs

Reply:
I agree, removed